internal/controller/services/auth/resources/admingroup-clusterrole.tmpl.yaml

```diff
@@ -2,6 +2,16 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: admingroupcluster-role
+  labels:
+    app.kubernetes.io/managed-by: opendatahub-operator
+    platform.opendatahub.io/part-of: auth
+  {{- if .DSCI }}
+  ownerReferences:
+  - apiVersion: {{.DSCI.APIVersion}}
```

Comment:
Why are we adding these ownerReferences?

Reply:
so it gets deleted along with the dsci